geode-membership/src/main/java/org/apache/geode/distributed/internal/membership/gms/GMSMembership.java

```diff
@@ -479,6 +476,9 @@ public void processView(final MembershipView<ID> newView) {
 
       // the view is complete - let's install it
       latestView = newlatestView;
+      for (ID newMember : newMembers) {
```

Comment:
I was just wondering if the lister code to trigger view installed be called before the new members connected event?